asserts/asserts.go

```diff
@@ -99,7 +99,21 @@ type Ref struct {
 }
 
 func (ref *Ref) String() string {
-	return fmt.Sprintf("%s %v", ref.Type.Name, ref.PrimaryKey)
+	pkStr := ""
+	n := len(ref.Type.PrimaryKey)
+	if n != len(ref.PrimaryKey) {
+		pkStr = "???"
+	} else if n > 0 {
```

Comment:
I don't think `n == 0` is ever going to happen IRL, so I think I'd treat that as a broken pk too, i.e. "???".